### DISCUSSION TITLE
to create the account for users to only view the photo use these cmd

### DISCUSSION THREAD

**Discussion by yashu1wwww** [photoprism/photoprism#4001] [General]
root to photoprism

then paste these cmd

photoprism users add yashu -p Yashu123 -n "yashu" -- role viewer

here yashu-username
Yashu123-password
"yashu-name

replace with your needed

**Comment by graciousgrey** [comment]
Please see our docs or all available user commands: https://docs.photoprism.app/user-guide/users/cli/

**Reply by yashu1wwww** [reply]
To change title and site author in photoprism docker we must have membership to it or free

Can you give full line cmd to change site title

**Reply by graciousgrey** [reply]
You can change the site title in all PhotoPrism editions by using the respective variable --> https://docs.photoprism.app/getting-started/config-options/#site-information

**Reply by yashu1wwww** [reply]
I asked it only for membership people's are free for all in docker photoprism
